Landelijke_uitslag_kiesmannen.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
```

Comment:
Ik dacht dat je aparte file als in functie in een aparte file bedoelde maar wel aan het project toegevoegd bedoelde